### PR TITLE
fix minor typo, remove comments on pip install

### PR DESCRIPTION
**Pull request by seungwonpark**
Hi, thanks for open-sourcing DDSP!

I've found out some typos and a comment that can be fixed right now.
Hope this helps.

P.S. I've signed to CLA as required in CONTRIBUTING.md.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # How to Contribute
 
 We'd love to accept your patches and contributions to this project.
-DSP can be subtle to get completely right, so we particularly appreciate the
```
Comment (adarob) [inline CONTRIBUTING.md]: DSP is actually intended here.

Comment (seungwonpark) [reply CONTRIBUTING.md]: Reverted to DSP in https://github.com/magenta/ddsp/pull/1/commits/13c29ccbb56a5431eaa424bc79bf1bfa77898a76. Thank you!

**Comment by adarob** [comment]
@googlebot add label

**Comment by adarob** [comment]
@googlebot add label

**Comment by adarob** [comment]
Working out some kinks in our import system but hoping to get this in today. Thanks!

**Pull request merged**